content/post/cheers-to-small-beginnings.md

```diff
@@ -0,0 +1,55 @@
+---
+author: ricardobaeta
+date: 2018-07-09
+title: "Cheers to small beginnings!"
+image: /post/cheers-to-small-beginnings/{d}ali-banner.png
```

Comment:
having braces in the path name seems risky hehe, maybe rename it to dali-banner?

Reply:
Thanks!